posthog/models/person.py

```diff
@@ -50,17 +50,17 @@ def merge_people(self, people_to_merge: List["Person"]):
 
             capture_internal(event, self.distinct_ids[-1], None, None, now, now, self.team.id)
 
-    def split_person(self, main_distinct_id: str):
-        with transaction.atomic():
-            distinct_ids = Person.objects.get(pk=self.pk).distinct_ids
-            if not main_distinct_id:
-                self.properties = {}
-                self.save()
-                main_distinct_id = distinct_ids[0]
-
-            PersonDistinctId.objects.filter(person=self).exclude(distinct_id=main_distinct_id).delete()
-            for distinct_id in distinct_ids:
-                if not distinct_id == main_distinct_id:
+    def split_person(self, main_distinct_id: Optional[str]):
+        distinct_ids = Person.objects.get(pk=self.pk).distinct_ids
+        if not main_distinct_id:
+            self.properties = {}
+            self.save()
+            main_distinct_id = distinct_ids[0]
```

Comment:
Trying to wrap my head around what happens if there's no `main_distinct_id`... and it isn't obvious from the method name nor this dialog:

![image](https://user-images.githubusercontent.com/53387/133674731-a1d3d096-f5cb-415d-9074-89ffa8430811.png)

... that if you don't specify an ID, we delete all properties of this user. I'd intuitively expect them to remain.

Could this behaviour be clarified in the dropdown? And in the code at least with a `"""` comment.